Python/Movie-Info-Telegram-Bot/README.md

```diff
@@ -0,0 +1,44 @@
+# Movie Info Telegram Bot
+
+## Description
+A telegram Bot made using python which scrapes IMDb website and has the following functionalities
+1. Replies to a movie name with genre and rating of the movie
+2. Replies to a genre with a list of top movies and tv shows belonging to that genre
+
+## Setup Instructions
+1. Create a bot in telegram:
+
+       1. Go to @BotFather and click /start and type /newbot and give it a name. 
+       2. Choose a username and get the token 
+       3. Place the token in "YOUR-TOKEN" place in the code
+
+2. Install required packages:
```

Comment:
Make a file `requirements.txt`, and mention the dependencies there.

Reply:
Sure, will work on it .Thank you